src/content/docs/en/guides/internationalization.mdx

```diff
@@ -255,7 +255,7 @@ This routing option allows you to customize your domains on a per-language basis
 
 Add `i18n.domains` to map any of your supported `locales` to custom URLs:
 
-```js title="astro.config.mjs" {3-7} ins={14-21}
+```js title="astro.config.mjs" {3-7} ins={14-19}
```

Comment:
Oh, more lines had to be deleted!
```suggestion
```js title="astro.config.mjs" {3-7} ins={14-17}
```